src/main/java/fr/xephi/authme/service/bungeecord/BungeeSender.java

```diff
@@ -75,13 +75,17 @@ public void connectPlayerOnLogin(Player player) {
      * @param type       The message type, See {@link MessageType}
      * @param playerName the player related to the message
      */
-    public void sendAuthMeBungeecordMessage(String type, String playerName) {
+    public void sendAuthMeBungeecordMessage(MessageType type, String playerName) {
         if (isEnabled) {
-            if(!plugin.isEnabled()) {
+            if (!plugin.isEnabled()) {
                 ConsoleLogger.debug("Tried to send a " + type + " bungeecord message but the plugin was disabled!");
                 return;
             }
-            sendBungeecordMessage("AuthMe", type, playerName.toLowerCase());
+            if (type.isBroadcast()) {
+                sendBungeecordMessage("Forward", "ALL", "AuthMe", type.getId(), playerName.toLowerCase());
+            } else {
+                sendBungeecordMessage("AuthMe", type.getId(), playerName.toLowerCase());
```

Comment:
This won't work in [1.13. Channels have to be in the following format:](https://hub.spigotmc.org/stash/projects/SPIGOT/repos/bukkit/browse/src/main/java/org/bukkit/NamespacedKey.java#32)
Namespace, Key := [a-z0-9._-]
Channel := Namespace:Key

and for 1.7.X support [smaller than 16 characters in total](https://github.com/bergerkiller/CraftSource/blob/8e1126adbbde1096b8351650bdd2c1ca0f6a9e46/org.bukkit/plugin/messaging/Messenger.java#L21).